src/Sarif/Core/WebRequest.cs

```diff
@@ -6,14 +6,27 @@
 using System.Linq;
 using System.Text;
 using System.Text.RegularExpressions;
-using Newtonsoft.Json;
 
 namespace Microsoft.CodeAnalysis.Sarif
 {
     public partial class WebRequest
     {
-        [JsonIgnore]
-        public string HttpVersion { get; private set; }
```

Comment:
>HttpVersion [](start = 22, length = 11)

I had added this convenience property, which the `Parse` method populated. I had marked it `JsonIgnore` because we can't serialize it because the SARIF spec doesn't define this property. The result is that when you serialize a `WebRequest` object containing this property, and then you deserialize the resulting string back into a `WebRequest` object, this property is lost. This made it confusing and not useful.